frontend/src/workStation/WorkStation.jsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import Nav from "../app/Nav";
+import Footer from "../app/Footer";
+import CodeIDE from "./CodeIDE";
+import { useParams } from "react-router-dom";
+import { Box, Text } from "@chakra-ui/react";
+
+function WorkStation() {
+  const [groupData, setGroupData] = useState([]);
+  const [fileData, setFileData] = useState([]);
+  const [error, setError] = useState("");
+  const { groupId, fileId } = useParams();
+
+  const BACKEND_URL = import.meta.env.VITE_BACKEND_URL;
+  const token = localStorage.getItem("jwt");
+
+  const fetchGroupDetails = async () => {
+    try {
+      const response = await axios.get(`${BACKEND_URL}/api/group/${groupId}`, {
+        headers: {
+          Authorization: `Bearer ${token}`,
+        },
+        withCredentials: true,
+      });
+      setGroupData("");
+      setGroupData(response.data);
+    } catch (error) {
+      setError("Error fetching group details");
+    }
+  };
+  const fetchFileDetails = async () => {
+    try {
+      const response = await axios.get(
+        `${BACKEND_URL}/group/${groupId}/files/${fileId}/get-file-details`,
+        {
+          headers: {
+            Authorization: `Bearer ${token}`,
+          },
+          withCredentials: true,
+        }
+      );
+      setFileData("");
+      setFileData(response.data);
+    } catch (error) {
+      setError("Error fetching file details");
+    }
+  };
+
+  useEffect(() => {
+    fetchGroupDetails();
+    fetchFileDetails();
+  }, []);
+
+  return (
+    <Box id="WorkStation">
+      <Text>Welcome to {groupData.groupName} Code Group</Text>
+      <Nav />
+      <Box>
+        <CodeIDE fileContent={fileData.fileContent} />
+      </Box>
+      <div>This is group: {groupId}</div>
```

Comment:
Nit: Normally we don't expose data IDs to the user so clearly, especially if the user has no use for them.

Reply:
Okay thanks. I have updated this in the newest commit in this branch. Here: https://github.com/izuchukwumba/peered/pull/7/commits/1d04a3e6e4fc6218291de43471fdebfdc707b1d1